MuvrKitTests/MKMarkovPredictorTests.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import XCTest
+@testable import MuvrKit
+
+class MKMarkovPredictorTests : XCTestCase {
+
+    
+    func testSequence(sequence: [Double], forExerciseId exerciseId: String, desiredValue: Double) {
+        let predictor = MKMarkovPredictor()
+        predictor.trainPositional(sequence, forExerciseId: exerciseId)
+        XCTAssertEqual(predictor.predictScalarForExerciseId(exerciseId, n: sequence.count), desiredValue)
+    }
+    
+    func testPrediction() {
```

Comment:
This is a good test for the entire sequence, but the actual usage in the gym is

[10] -> expect 15 (but only reasonable guess is 10)
[10, 15] -> expect 20
[10, 15, 20] -> expect 25
[10, 15, 20, 25] -> expect 30
[10, 15, 20, 25, 25] -> expect 25 or 20
[10, 15, 20, 25, 25, 20] -> expect 15 or 20 (or suggest _no more_ / _done_)